index.html

```diff
@@ -223,52 +141,84 @@ <h3>Skills</h3>
     <section class="section-contact" id="contact-section">
       <div class="desktop-contact d-flex">
         <p class="contact-desc">
-          I'm always interested in hearing about new projects, so if you'd like to
-          chat please get in touch.
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
         </p>
-        <form class="contact-form" action= "https://formspree.io/f/xqkoyalw" method="post">
+        <form
+          class="contact-form"
+          id="form"
+          action="https://formspree.io/f/xqkoyalw"
+          method="post"
+        >
           <label for="firstname">First name: </label>
```

Comment:
- [ ] Great job working on this milestone. Validation functionalities work perfect. Unfortunately, you have issues with the figma design.
- In the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1-468) there is first name and last name, but your design has full name. Kindly fix this as project requirements says your contact form should look exactly like the figma design
**Figma design**
![image](https://user-images.githubusercontent.com/62849872/232164440-8bc4e1af-504d-479d-a968-7789f0b09b71.png)

Reply:
Thanks for the commendation. 
I don't think the issue you raised here is necessary because this issue has been addressed in the requirement for this section
![Screenshot from 2023-04-15 13-04-40](https://user-images.githubusercontent.com/89491845/232227427-308367f5-ad98-4902-880c-fe1d3697532a.png)